manuscript/algorithm.tex

```diff
@@ -181,3 +190,68 @@ \subsubsection{\gls{MARKAL}}
 of all the coefficients for the \gls{LP} problem, and is
 solved. 
 
+
+\subsection{Stochastic-Optimizing (SO)}
+Stochastic optimization refers to optimization methods
+that incorporate randomness in either the objective function
+or the constraints \cite{lauren_stochastic_2015}. It aims to find
+the roots for the objective function by generating random variables.
+This makes it fit for problems that contain uncertainty, which
+real world problems mostly are. 
+The random variables are defined by a probability distribution
+or sometimes just a few values.
+
+Mathematically, the method attempts to find properties of $f$ without
+evaluating $f$ directly, but by using random samples of $F(\theta, \xi).$
+
+\begin{align}
+	f(\theta)= E_{\xi} [F(\theta, \xi)] \\
+	\xi = \mbox{random variable} \\
+	f(\theta) = \mbox{objective function} \\
+	F(\theta,\xi) = \mbox{ random samples created} \\
+\end{align}
+
+A few examples of the Stochastic-Optimizing method are
+the Markov Switching Model and Gaussian Process Regression.
+
+
+\subsubsection{Markov Switching Model}
+The Markov Switching Model depends on the idea that
+the future is independent of the past and only dependent
+on the present. It utilizes Markov Chains, which 
+characterize the probability of a system transitioning
+between states."
+
+The Markov Model is in a wide variety of applications,
+such as predicting exchange rates \cite{engel_can_1994}, 
+labor markets \cite{krolzig_a_2002} to search trends \cite{bergamaschi_a_2011}.
+
+The Markov Switching Model uses two types of equations:
+measurement and transition equations \cite{ansari_predicting_2015}.
+The measurement equations determine how hidden states affect observed data,
+and transition equations define how the states evolve over time.
+The model is represented with the following equations:
+
+\begin{align}
+	y_t =a_{0,0} + a_{0,1} S_t + (a_{1,0}+a_{1,1} S_t) y_{t-1} +e_i \\
+	P_{ij}=P(S_t=j | S_{t-1} = i) \\
+	y_t = \mbox{observable random variable} \\
+	a = \mbox{drift term constant} \\
+	S_t = \mbox{hidden state variable} \\ 
+	e_i = \mbox{error} \\
+	P_{ij} = \mbox{probability of a transition in the hidden state variable} \\ 
+	
+\end{align}
+
+
+\subsection{Gaussian Process Regression}
+Gaussian Process Regression distributes a function as a Gaussian Process
+characterized by a mean function and a covariance function \cite{melo_gaussian_2012}.
```

Comment:
Here are the bib warnings I get (for example, please note that Lauren Hannah has the last name Hannah, not the last name "Lauren")

```
huff@lise:~/repos/ddca-litrev/manuscript((HEAD detached at jbae11/master)) $ bibtex ddca-litrev
This is BibTeX, Version 0.99d (TeX Live 2015/Debian)
The top-level auxiliary file: ddca-litrev.aux
The style file: elsarticle-num.bst
Database file #1: ddca-litrev.bib
Warning--I didn't find a database entry for "lauren_stochastic_2015"
Warning--I didn't find a database entry for "krolzig_a_2002"
Warning--I didn't find a database entry for "bergamaschi_a_2011"
Warning--I didn't find a database entry for "yáñez_agent-based_2009"
Warning--empty pages in carre_overview_2009
Warning--empty journal in kavei_application_2008
Warning--empty journal in focardi_arch/garch_2007
Warning--empty school in schweitzer_improved_2008
Warning--empty pages in gonzalez_detailed_2002
(There were 9 warnings)
```